testing/src/main/java/org/oppia/android/testing/RecyclerViewTesting.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.android.testing
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import org.oppia.android.testing.RecyclerViewMatcher.Companion.atPositionOnView
+
+class RecyclerViewTesting {
```

Comment:
please take reference of `EditTextInputAction`. We don't need `companion object`.

Reply:
Also add kdoc for this class

Reply:
Done